signup/tool/src/java/org/sakaiproject/signup/tool/downloadEvents/EventWorksheet.java

```diff
@@ -56,7 +56,7 @@
 import org.apache.poi.ss.usermodel.Workbook;
 import org.apache.poi.ss.util.CellRangeAddress;
 import org.apache.poi.xssf.usermodel.XSSFWorkbook;
-
+import org.sakaiproject.component.cover.ServerConfigurationService;
```

Comment:
Please you spring DI instead of covers but If you need to use a cover use the ComponentManager.get().

Reply:
Use the org.sakaiproject.signup.logic.SakaiFacade class as that already has all dependencies and you can either create a new method for getting this config, or get the ServerConfigurationService directly.

Reply:
Thank you for the review! Requested changes made.